templates/joomla/css/custom.css

```diff
@@ -18,20 +18,7 @@
 }
 
 /*Partners page - anchors 2018*/
-#algolia:target,
-#rf:target,
-#cleantalk:target,
-#cloudaccess:target,
-#crowdin:target,
-#github:target,
-#glip:target,
-#hostingde:target,
-#jetbrains:target,
-#ost:target,
-#rochen:target,
-#screaming:target,
-#sitelock:target,
-#sflc:target {
+.anchors {
```

Comment:
Probably need the `:target` pseudo-class back here 😉

Reply:
Pfff. I need to sleep. 
Anyway, that was working fine with # only, now I have 2 which are screwed, don't know why. I will try to understand why.